plugins/CryptMessage/CryptMessagePlugin.py

```diff
@@ -200,8 +187,8 @@ def testCryptEciesDecrypt(self, num_run=1):
         aes_key, encrypted = CryptMessage.eciesEncrypt(self.utf8_text.encode("utf8"), self.publickey)
         for i in range(num_run):
             assert len(aes_key) == 32
-            ecc = CryptMessage.getEcc(self.privatekey)
-            assert ecc.decrypt(encrypted) == self.utf8_text.encode("utf8"), "%s != %s" % (ecc.decrypt(encrypted), self.utf8_text.encode("utf8"))
+            decrypted = CryptMessage.eciesDecrypt(base64.b64encode(encrypted))
```

Comment:
missing privatekey argument, it should be: `decrypted = CryptMessage.eciesDecrypt(base64.b64encode(encrypted), self.privatekey)`

Reply:
There, fixed.